queries.yaml

```diff
@@ -1841,6 +1841,38 @@ post_process:
       # need to be a large tolerance.
       simplify_tolerance: 1.0
 
+  # drop labels on roads which are too short to render.
+  - fn: vectordatasource.transform.drop_names_on_short_boundaries
+    params:
+      source_layer: roads
+      start_zoom: 8
+      end_zoom: 15
```

Comment:
Since tile size is not our only measure: for `13/4089/2723.mvt` (512 px size) we see several hundred names being dropped in central London, which is great! This means the client doesn't need to do the CPU work of trying to layout names that can't fit anyhow.

We don't do this for zoom 14 at 512 px because that's nominal zoom 15 (changing to `end_zoom: 16`). If we wanted to be super aggressive we could do it at zoom 15 and not at nominal zoom 16. Looking at QGIS at `14/8189/5446.mvt` I see 945 road features of which 665 have names. But probably 500 of those could be dropped because of their short length.

We'd also need apply `end_zoom: 16` below... and consider it above as well.

Reply:
with the new build after commit 2e14e61, the 14/8189/5446 tile doesn't change.